EXPLAINER_fullscreen_popups.md

```diff
@@ -0,0 +1,161 @@
+*   Financial app opens a fullscreen dashboard on the primary monitor and a fullscreen stock tracker window on a secondary display.
+*   Virtual desktop app launches fullscreen windows on primary and secondary displays to mimic a remote monitor configuration.
+
+Furthermore, a web application could launch N fullscreen popup windows, as long as the user agent allows N popups from a user gesture (i.e. multiple popups allowed by popup blocker being disabled):
+*   Security monitoring app launches 6 fullscreen video feeds on an array of 6 displays.
+*   Gaming app launches 3 fullscreen windows across 3 displays to provide a continuous widescreen view.
+
+## Proposal
+Allow sites with the `window-management` permission to open a fullscreen window on a specified display from a single user gesture. Scripts calling  <code>[window.open()](https://html.spec.whatwg.org/multipage/window-object.html#dom-open-dev)</code> to request a popup window could also include a new <code>fullscreen</code> boolean window feature.
+
+Fullscreen popup requests could also include window bounds features (e.g. `left`, `top`, `width`, `height`), with positions specified [relative to the multi-screen origin](https://w3c.github.io/window-placement/#api-window-attribute-and-method-definition-changes), to request that the window be made fullscreen on the screen containing those bounds, and also to be used as the popup window bounds after exiting fullscreen.
+
+### Example Code
+
+#### Basic Example:
+```js
+window.open(url, '_blank', 'popup,fullscreen');
+```
+
+#### Opening a fullscreen popup on another display:
+```js
+const otherScreen = screenDetails.screens.find(s => s !== screenDetails.currentScreen);
+window.open(url, '_blank', `left=${otherScreen.availLeft},` +
+                            `top=${otherScreen.availTop},` +
+                            `width=${otherScreen.availWidth},` +
+                            `height=${otherScreen.availHeight},` + 
+                            `fullscreen`);
+```
+
+### Spec Changes
+The spec does not contain an exhaustive list of valid features in the `features` argument, however, the spec does contain procedures for determining if a [popup window is requested](https://html.spec.whatwg.org/multipage/window-object.html#popup-window-is-requested), and the specific `tokenizedFeatures` to look for. Following this model, the spec shall be updated to define a similar algorithm for checking if a **fullscreen window** is requested. A fullscreen window must also first be considered a popup window.
+
+After [checking if a popup window is requested](https://html.spec.whatwg.org/multipage/window-object.html#popup-window-is-requested), add the following section:
+
+To **check if a fullscreen window is requested**, given _tokenizedFeatures_:

```

Comment:
Keep this in sync with more updated discussions and documents (include fullscreen permission policy check).

Reply:
See above, I missed merging a commit that adjusted all of this.